docs/node-mixin/alerts/add-runbook-links.libsonnet

```diff
@@ -0,0 +1,23 @@
+local utils = import '../lib/utils.libsonnet';
+
+local lower(x) =
+  local cp(c) = std.codepoint(c);
+  local lowerLetter(c) =
+    if cp(c) >= 65 && cp(c) < 91
+    then std.char(cp(c) + 32)
+    else c;
+  std.join('', std.map(lowerLetter, std.stringChars(x)));
+
+{
+  _config+:: {
+    runbookURLPattern: 'https://github.com/prometheus/node_exporter/tree/master/docs/node-mixin/runbook.md#alert-name-%s',
```

Comment:
We don't have this file yet. Perhaps we should at least add a skeleton with each alert anchored, even if each "runbook" will just be a TODO for now?